types/react-dom/index.d.ts

```diff
@@ -19,6 +19,7 @@ import {
 
 export function findDOMNode(instance: ReactInstance | null | undefined): Element | null | Text;
 export function unmountComponentAtNode(container: Element): boolean;
+export function renderToStaticMarkup(element: Element): string;
```

Comment:
I don't believe this is the correct place for this export based on the provided documentation. It seems it is already provided in 'react-dom/server': https://github.com/DefinitelyTyped/DefinitelyTyped/blob/efd47a5cf2befc4c04c7f2fbe3f69b31f3d71857/types/react-dom/server/index.d.ts#L30-L36